### PR TITLE
Fixes for various errors

### PR DESCRIPTION
**Pull request by seba-aln**
fix: Fix typo in consumers v3 

This change fixes typo in consumer models user and space resulting in setting invalid flags for the request.

fix: Fix method `is_error` in common model

This change fixes error in calling and returning value of `status.is_error()` method 

fix: Add misc informations to PyPi package

This change adds additional informations to PyPi package. Informations include URLs to source code and documentation, required python version (at least 3.7) and updates a list of supported python versions (removed 3.6 and added 3.10)

Closes #145 
Closes #102 
Closes #115

### CODE REVIEW

File: pubnub/models/consumer/common.py [hunk 1]
```diff
@@ -20,4 +20,4 @@ def __init__(self):
         self.affected_groups = None
 
     def is_error(self):
-        return self.error is not None
+        return True if self.error else False
```
Comment (marek-lewandowski) [inline pubnub/models/consumer/common.py:23]: You can also try with:
```
return bool(self.error)
```
It should return the same results ;)

File: setup.py [hunk 2]
```diff
@@ -7,22 +7,26 @@
...
+        'Documentation': 'https://www.pubnub.com/docs/sdks/python',
+    },
     packages=find_packages(exclude=("examples*", 'tests*')),
     license='MIT',
     classifiers=(
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
```
Comment (marek-lewandowski) [inline setup.py:17]: Based on `python_requires` this package should work with Python 3.7

**Comment by seba-aln** [comment]
@client-engineering-bot release

**Comment by pn-op** [comment]
🚀 Release successfully completed 🚀

**Pull request merged**